pilot/pkg/features/experimental.go

```diff
@@ -197,4 +197,7 @@ var (
 	EnableLocalityWeightedLbConfig = env.Register("ENABLE_LOCALITY_WEIGHTED_LB_CONFIG", false,
 		"If enabled, always set LocalityWeightedLbConfig for a cluster, "+
 			" otherwise only apply it when locality lb is specified by DestinationRule for a service").Get()
+
+	UnifiedSidecarScoping = env.Register("PILOT_UNIFIED_SIDECAR_SCOPE", true,
```

Comment:
PILOT_UNIFIED_SIDECAR_SCOPE doesnot reflect the intention of the feature.  The intention is first choose proxy local kubernetes services

Reply:
Its changing more than just that. I don't really know a better way to convey that. We can call it "enhanced sidecar scope" if you prefer I guess..?

FWIW this is really an internal flag so I don't care about the name too much... if you have a suggestion I am happy to call it anything